File: src/main/java/com/eucaly/HelloWorld.java
package com.eucaly;

public class HelloWorld {
    public static void main(String[] args) {
//        System.out.println("Hello World");
        Person p = new Person("Eucaly", 45.7f, 1.56f);
        p.hello();
        p.weight = 45.5f;
        p.hight = 1.56f;
        System.out.println(p.bmi());
        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        byte b = 120;
        float weight = 44.8f;
        float height = 1.56f;
        boolean adult = true;
        boolean enroll = false;
        String name = "Eucaly";*/
    }
}

File: src/main/kotlin/com/eucaly/HelloWorld.kt
package com.eucaly

fun main(args : Array<String>) {
//    println("Hello Kotlin")
//    Human().hello()
    val h = Human(45.5f, 1.56f)
    println(h.bmi())
    /*var age = 19
    age = 20
    var weight = 44.8f
    var name : String
    name = "Eucaly"*/

}

class Human(var weight : Float, var height : Float, var name : String = "") {
    init {
        println("test $weight")
    }
//    constructor(name : String, weight : Float, height : Float) : this(weight, height)
    fun bmi() : Float {
        val bmi = weight / ( height * height)
        return bmi
    }
    fun hello() {
        println("Hello Kotlin.")
    }
}